app/src/main/res/layout/activity_login.xml

```diff
@@ -17,84 +17,86 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         android:src="@drawable/ic_logo_login"
-        android:layout_marginTop="10dp"
+        android:layout_marginTop="@dimen/main_activity_margin_top"
         android:contentDescription="@string/app_name"/>
 
     <TextView
         android:id="@+id/txt_error"
-        android:layout_width="300dp"
-        android:layout_height="25dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         app:layout_constraintTop_toBottomOf="@id/img_logo"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0"
         android:fontFamily="@font/roboto_bold"
         android:textColor="#CA2020"
-        android:textSize="13sp"
+        android:textSize="@dimen/sp13"
         />
     <TextView
         android:id="@+id/txt_user"
-        android:layout_width="100dp"
-        android:layout_height="25dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         app:layout_constraintTop_toBottomOf="@id/txt_error"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0"
         android:text="@string/user"
         android:fontFamily="@font/roboto_bold"
         android:textColor="@color/quaternary"
-        android:textSize="16sp"
+        android:textSize="@dimen/sp16"
         />
 
     <EditText
         android:id="@+id/input_user"
         android:layout_width="match_parent"
-        android:layout_height="49dp"
+        android:layout_height="@dimen/main_activity_height_input_btn"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks